app/models/dialog_field_serializer.rb

```diff
@@ -25,7 +25,7 @@ def serialize(dialog_field)
       category = Category.find_by(:id => dialog_field.category)
       dialog_field.options.merge!(:category_name => category.name, :category_description => category.description)
     end
-
+    return dialog_field.as_json(:methods => [:type, :values]).merge(extra_attributes) if all_attributes
```

Comment:
How do you feel about simplifying all of these return calls in the various serializers and just passing `all_attributes` down into `included_attributes`? And then `included_attributes` in the `Serializer` superclass would just have a `return attributes if all_attributes?` clause?